packages/components/src/internal/components/entities/actions.ts

```diff
@@ -462,7 +466,7 @@ export function getEntityTypeData(
             getEntityTypeOptions(entityDataType),
             // get all the parent schemaQuery data
             getChosenParentData(model, parentSchemaQueries, allowParents, isItemSamples),
-            ...parentSchemaQueries.map(getEntityTypeOptions).toArray(),
+            ...parentSchemaQueries.map(edt => getEntityTypeOptions(edt)).toArray(),
```

Comment:
was this a linting change? just curious why this `edt` would need to be added here?

Reply:
This was important. I changed the arguments to `getEntityTypeOptions` and this changes how the function gets mapped. I would say this is more of a problem with JS in general. That said, to get around it, you need to explicitly pass the arguments.